pkg/security/secl/model/process_cache_entry.go

```diff
@@ -214,7 +215,7 @@ func (p *ArgsEnvsCacheEntry) toArray() ([]string, bool) {
 
 	for entry != nil {
 		v, err := UnmarshalStringArray(entry.ValuesRaw[:entry.Size])
-		if err != nil || entry.Size == 128 {
+		if err != nil || entry.Size == 256 {
```

Comment:
Should we parameterize it ? Just to make sure we keep this value in sync with what we unmarshall in unmarshallers.go